staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go

```diff
@@ -143,14 +143,29 @@ func ruleMatchesResource(r *audit.PolicyRule, attrs authorizer.Attributes) bool
 
 	apiGroup := attrs.GetAPIGroup()
 	resource := attrs.GetResource()
+	fullResource := ""
+	if sr := attrs.GetSubresource(); sr != "" {
+		fullResource = resource + "/" + sr
+	}
+
+	name := attrs.GetName()
+
 	for _, gr := range r.Resources {
 		if gr.Group == apiGroup {
 			if len(gr.Resources) == 0 {
 				return true
 			}
-			for _, res := range gr.Resources {
-				if res == resource {
-					return true
+			for _, r := range gr.Resources {
+				// Resource either matches the resource directly or matches the
+				// resource + the subresource.
+				//
+				// 		"pods" matches "pods" and "pods/logs"
+				//		"pods/logs" only matches "pods/logs"
+				//
+				if r == resource || (fullResource != "" && r == fullResource) {
```

Comment:
Let's consider the user perspective. A new user is trying to set up audit logging. The want to log pod requests, so they do:
```yaml
policy:
- level: Request
  resources: ["pods"]
```
Do you think they would expect a request to `pods/status` to also be logged? I think the answer is probably yes...